test/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "test",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "test": "jest --runInBand"
```

Comment:
Would it be helpful for a `test-debug` script here, to config the debugging envvar?